### ISSUE TITLE
I use 300 pictures to finetune,but the result is bad!

### ISSUE THREAD

**Issue by Flyooofly** [amirbar/DETReg#27]

**Comment by amirbar** [comment]
Perhaps try a longer training schedule without a learning rate drop. Specifically, try setting `lr_drop` to be the number of epochs at least on the first attempt.

**Comment by luoyq6** [comment]
> _未提供描述。_
Excuse me, hello. Why do I get the error shown in the picture when I use the command to evaluate? Could you give me some tips?
![b39205ed6c3af39c07dc62491331797](https://user-images.githubusercontent.com/110521846/218690468-9c5dd5c7-dc0c-432f-9cf2-28980b059fbb.png)
